projects/storefrontapp-e2e-cypress/package.json

```diff
@@ -2,19 +2,22 @@
   "name": "storefrontapp-e2e-cypress",
   "version": "1.0.0",
   "main": "index.js",
-  "author": "Marcin Lasak",
+  "author": "Spartacus team",
   "license": "MIT",
   "scripts": {
     "cy:open": "cypress open",
     "cy:run": "cypress run",
-    "cy:run:ci": "cypress run --config-file cypress.ci.json --spec \"cypress/integration/!(vendor)/**/*.e2e-spec.ts\"",
-    "cy:run:ci:all": "cypress run --config-file cypress.ci.json",
-    "cy:run:smoke": "cypress run --spec \"cypress/integration/smoke/**/*\"",
-    "cy:run:smoke:ci": "cypress run --config-file cypress.ci.json --spec \"cypress/integration/smoke/**/*\"",
+    "cy:run:ci:1905": "cypress run --config-file cypress.ci.1905.json --spec \"cypress/integration/!(vendor)/**/*.e2e-spec.ts\"",
```

Comment:
should we have a cy:run:ci as well like mobile, regression, and smoke variants

Reply:
but the shortcut was removed in the package.json global level, so I'm guessing we don't want it

Reply:
Yes, we don't really want it.